examples/example1.py

```diff
@@ -72,10 +73,10 @@ def get_transcript(name):
 # hgvs_name.ref_allele = 'A'
 # hgvs_name.alt_allele = 'G'
 
-print (hgvs_name.transcript,
+print((hgvs_name.transcript,
```

Comment:
This is a bit of a nit, and I realize this is technically valid Python 2 code, but to guarantee compatibility between Python 2.7 and Python 3, could you add a `from __future__ import print_function` to the top of files that use `print` as a function?